src/sentry/api/serializers/models/organization.py

```diff
@@ -135,8 +136,9 @@ def serialize(self, obj, attrs, user):
                 # This feature_name was found via `batch_has`, don't check again using `has`
                 org_features.remove(feature_name)
 
+        # Remaining features should not be checked via the entity handler
         for feature_name in org_features:
-            if features.has(feature_name, obj, actor=user):
+            if features.has(feature_name, obj, skip_entity=True, actor=user):
```

Comment:
nit: keep the order as the definition so `features.has(features_name, obj, actor=user, skip_entity=True)`